src/audio/volume/volume.c

```diff
@@ -344,6 +345,19 @@ static struct comp_dev *volume_new(const struct comp_driver *drv,
 	if (!cd)
 		goto fail;
 
+	/* malloc memory to store current volume 4 times to ensure the address
+	 * is 8-byte aligned for multi-way xtensa intrinsic operations.
+	 */
+	cd->vol = rzalloc(SOF_MEM_ZONE_RUNTIME, 0, SOF_MEM_CAPS_RAM, vol_size);
```

Comment:
There is an aligned malloc() that we should use and it guarantees alignment.

Reply:
@lgirdwood  this one ? cd->vol = rballoc_align(0, SOF_MEM_CAPS_RAM, vol_size, 8);